andela/wg-7-horsemen
# #152919280 Upgrade the nutrition diary feature

#### What does this PR do?
Currently wger provides a user with the ability to create a nutrition plan for the day but does not provide functionality for a user to track what they actually ate. This PR provides functionality for a logged in user to track what they actually ate along side what they planned.

#### Description of Task to be completed?
- Provide features for a user to create, edit and delete the meal they actually ate.
- Display the meals they actually consumed alongside what the user planned to eat.

#### How should this be manually tested?
- After setting up the application. Navigate to the 'Nutrition' section to create a new nutrition plan.
- To the nutrition plan, add a meal and add items to it.
- Hover to the 'consumed' column row corresponding to the meal scheduled. Click the '+' button and add the meal consumed.
- On saving the form, the meal consumed should appear on the list along side the planned meal.

#### What are the relevant pivotal tracker stories?
#152919280

#### Screenshots
<img width="1440" alt="screen shot 2017-12-03 at 20 53 13" src="https://user-images.githubusercontent.com/3184113/33529254-3bffc14e-d87f-11e7-86a3-9ec3685d0a4c.png">

<img width="1437" alt="screen shot 2017-12-03 at 20 52 06" src="https://user-images.githubusercontent.com/3184113/33529251-3b369a6c-d87f-11e7-8bc0-f7107a4e692f.png">

<img width="1440" alt="screen shot 2017-12-03 at 20 52 44" src="https://user-images.githubusercontent.com/3184113/33529252-3b7aef78-d87f-11e7-88f8-3a73bc65383b.png">

<img width="1438" alt="screen shot 2017-12-03 at 20 52 58" src="https://user-images.githubusercontent.com/3184113/33529253-3bbd008e-d87f-11e7-967a-b4a55b5b327e.png">
